src/test/kotlin/org/coepi/api/dao/ReportsDaoTest.kt

```diff
@@ -16,13 +22,49 @@ import java.util.*
  *
  * Until then, don't expect them to succeed, just use them as documentation
  */
+
 @Disabled
 class ReportsDaoTest {
 
     private val dao: ReportsDao = ReportsDao()
     val cenKeys = arrayOf("foo", "bar")
     val reportData = "foobar".toByteArray(Charset.defaultCharset())
 
+    companion object{
+
+        val LOCAL_URL= "http://localhost:9000";
+        val REGION = "us-west-2"
+
+        @BeforeAll
+        @JvmStatic
+        internal fun startDynamoDb(){
+            System.setProperty("sqlite4java.library.path", "./build/libs")
+            val server = ServerRunner.createServerFromCommandLineArgs(arrayOf("-inMemory", "-port", "9000"))
+            server.start();
+
+            val dynamoClient = AmazonDynamoDBClientBuilder.standard()
+                    .withEndpointConfiguration(AwsClientBuilder.EndpointConfiguration(LOCAL_URL, REGION))
+                    .build();
+            val dynamoDb = DynamoDB(dynamoClient)
+
+            val table = dynamoDb.createTable(
```

Comment:
You can use DDB Mapper and re-use the models to create table.

https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/dynamodbv2/datamodeling/DynamoDBMapper.html

Reply:
Converted the code to create table using DDB Mapper.